NetworkPkg/TcpDxe/TcpOption.c

```diff
@@ -154,7 +160,13 @@ TcpSynBuildOption (
              NET_BUF_HEAD
              );
 
-    ASSERT (Data != NULL);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (Data == NULL) {
+      ASSERT (Data != NULL);
+      return 0; // Returning Len of -1 if we fail allocating space

```

Comment:
Other comments are updated to `0`, but this one is still `-1`.

Reply:
Fixed.